DataFormats/GEMRecHit/src/GEMSegment.cc

```diff
@@ -10,14 +10,7 @@ namespace {
   // create reference GEM Chamber ID for segment
   inline
   DetId buildDetId(GEMDetId id) {
-    // GE1/1 => station 1
-    // GE2/1 => station 2 & 3 => combine Short & Long to mimick future technology that can have 4+ layers
-    int station, layer;
-    if (id.station()==1) station = 1;
-    else if (id.station()==2 || id.station()==3) station = 3;
-    else station = 0; 
-    layer = 0;
-    return GEMDetId(id.region(),id.ring(),station,layer,id.chamber(),id.roll());
+    return GEMDetId(id.region(),id.ring(),id.station(),0,id.chamber(),id.roll());
```

Comment:
Yes, Relvals made with earlier versions of 81X will not be readable with the next prerelase (when this PR will be in). But I suppose that is not a problem? 

Reason for this fix:
Since 80X the  station 2 (GE2/1 short) is not in the geometry anymore. Now in a past PR (during 81X development) changes were made to rename station 3 --> station 2. This is one of the places where the change was forgotten, giving rise to problems in the Validation step of RunTheMatrix.